DatadogRUM/Sources/Instrumentation/AppHangs/AppHangsObserver.swift

```diff
@@ -1,85 +0,0 @@
-/*
```

Comment:
git couldn't resolve the merge conflict and let it be there while this file has been removed from develop branch.

Hence removed it manually.